docs/source/topics/ref_dns-configuration.adoc

```diff
@@ -17,7 +17,29 @@ Additional checks are done to verify DNS is properly configured when running [co
 [id="dns-configuration-linux_{context}"]
 == Linux
 
-On Linux, {prod} expects the following DNS configuration:
+On Linux, depending on your distribution, {prod} expects the following DNS configuration.
+
+=== NetworkManager + systemd-resolved
+
+This configuration is used on Fedora 33 or newer, and on Ubuntu Desktop editions.
+
+* {prod} expects NetworkManager to manage networking.
+* {prod} configures systemd-resolved  to forward requests for the `testing` domain to the `192.168.130.11` DNS server. `192.168.130.11` is the IP of the {prod} virtual machine.
```

Comment:
`systemd-resolved` should be formatted. This goes for the following line as well. :)

Also, could you please put each sentence on its own line? Thank you!

Reply:
`systemd-resolved` needs formatting, but not `NetworkManager`? I've made the change already, but looking back this seems inconsistent :)

Reply:
NetworkManager is the name of the project, so we don't format it. You might have a point regarding systemd-resolved and whether we should be formatting that or not -- I'm not sure!